tests/conftest.py

```diff
@@ -266,7 +259,9 @@ def hera_loc():
 
 @pytest.fixture(scope="session")
 def apollo_loc():
-    if hasmoon:
+    try:
+        from lunarsky import MoonLocation
+
         return MoonLocation(lat=0.6875, lon=24.433, height=0)
-    else:
+    except ImportError:
```

Comment:
Is there any advantage to the contextlib supression vs a try/except? the rest of the code base for this change seems to like the context lib. Maybe this try/except just reads easier for you in this fixture?

Reply:
Totally understandable, I asked because it was different from how the other import attempts were in this update.